src/ip.py

```diff
@@ -727,17 +727,24 @@ def do_neigh_flush(argv, af):
 def main(argv):
     # Detect Address family
     af = -1  # default / both
-    if argv and argv[0] == "-6":
+    if "-6" in argv:
         af = 6
-        argv.pop(0)
-    elif argv and argv[0] == "-4":
+        argv.remove("-6")
+    elif "-4" in argv:
         af = 4
-        argv.pop(0)
+        argv.remove("-4")
+    
+    # Ignore color option (-c --color)
+    for option in ["-c", "--c"]:
```

Comment:
```suggestion
    for option in ["-c", "--color"]:
```

`--color` doesnt match `--c`, it's exact string matching

Reply:
Added the suggestion for stderr, --c and -c also match (-)-c[olor=always|never|auto] as we are checking if the arguments starts with --c or -c.

Hope to have some time to think about the flag parsing in the near future.

Reply:
Not sure if I am missing something, but I already tested this. Below is an example. For you it didn't work?
<img width="911" alt="Screenshot 2023-01-13 at 13 32 21" src="https://user-images.githubusercontent.com/43178421/212321211-49674c0b-ce78-4c46-a0c4-dd90ad607714.png">

Reply:
Not sure if I am missing something, but I tested this and it worked. For you it didn't work?

Reply:
Ah no, but I see that only in the next few lines they're compared with startswith, which I missed